sklearn/ensemble/bagging.py

```diff
@@ -387,6 +404,38 @@ def _validate_y(self, y):
         # Default implementation
         return column_or_1d(y, warn=True)
 
+    def _get_estimators_data_draws(self, sample_mask=False, feature_mask=False):
+        # Get drawn indices or mask along both sample and feature axes
+        for seed in self._seeds:
+            # Operations accessing random_state must be performed identically
+            # to those in `_parallel_build_estimators()`
+            random_state = np.random.RandomState(seed)
+            feature_indices, sample_indices = _generate_bagging_indices(random_state, 
+                                                                  self.bootstrap_features,
+                                                                  self.bootstrap, 
+                                                                  self.n_features_, 
+                                                                  self._n_samples, 
+                                                                  self._max_features, 
+                                                                  self._max_samples)
+
+            # Convert indices to mask if desired
+            feature_draw = (_generate_mask_from_indices(feature_indices, self._n_samples)
+                            if feature_mask else feature_indices)
+            sample_draw = (_generate_mask_from_indices(sample_indices, self._n_samples)
+                           if sample_mask else sample_indices)
+
+            yield feature_draw, sample_draw
+
+    @property
+    def estimators_samples_(self):
+        """Return a dynamically generated list of boolean masks identifying 
```

Comment:
Cut the initial description back to one line, as in PEP257, then leave a blank line before a fuller description. Can copy the initial description from the Attributes section of the class docstring.